Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -587,22 +525,16 @@ namespace Babylon
 
     void NativeEngine::DeleteVertexArray(const Napi::CallbackInfo& info)
     {
-        delete info[0].As<Napi::External<VertexArray>>().Data();
+        auto vertexArray{info[0].As<Napi::External<VertexArray>>().Data()};
+        // TODO: should we clear the m_boundVertexArray if it gets deleted?
+        //assert(vertexArray != m_boundVertexArray);
```

Comment:
Can this be removed, or is this still a relevant question?

Reply:
I have a TODO in my first comment.